src/docs/reference/usage-limits.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage Limits
+---
+
+<PriorityBoardingBanner />
+
+In short, Usage Limits allow you to set a maximum limit on your usage for a single billing cycle. If your resource usage exceeds your limit, we will shut down your resources to prevent further usage.
+
+## Configuring usage limits
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1694775828/usage-limits_hzv9ee.png" alt="Usage Limits Modal" layout="responsive" width={1252} height={1150} />
+
+You can visit your [account usage page](https://railway.app/account/usage) to set the usage limits for your account. Once you click the `Set Usage Limits` button, you will see the modal above where you can set a `Custom email threshold` and a `Hard limit`.
+
+<Banner variant="info">The link above takes you to the usage page for your personal account. If you want to set a usage limit for your team, you can use the account switcher to access the team's usage.</Banner>
+
+## Custom email threshold
+
+You can think of this as a _soft limit_. When your resource usage crosses the specified amount, we will send you an email notifying you of the same. Your resources will remain unaffected.
+
+## Hard limit
+
+Think of the hard limit as the absolute maximum amount you're willing to spend on your infrastructure. As once your resource usage touches your hard limit, all your resources will be taken offline to prevent further usage.
```

Comment:
```suggestion
Once your resource usage hits the specified hard limit, all your workloads will be taken offline to prevent them from incurring further resource usage. Think of the hard limit as the absolute maximum amount you're willing to spend on your infrastructure.
```

Reply:
Fixed manually.